### DISCUSSION TITLE
[SOLVED] Find in all content of the pages

### DISCUSSION THREAD

**Discussion by cowpiebrownrice** [requarks/wiki#5813] [Help / Questions] [answered]
Hi to all, it's possible to extend find in all content of the pages?

Now find is only in title and description if I'm not mistaken

Thanks in advance

**Comment by cowpiebrownrice** [comment, marked as answer]
i've activated "Search Engines --> Database - PostgreSQL" and now i can find in all text of the pages!

**Reply by JanZeman** [reply]
Hi congrats to that! My database is MariaDB. Does it mean I cannot have the content search?

**Reply by NGPixel** [reply]
You need to use one of the external search engines if using MariaDB.

**Reply by JanZeman** [reply]
I see. Thank you for the explanation, @NGPixel
